src/pages/messages/components/TicketList.tsx

```diff
@@ -41,10 +34,14 @@ export const TicketList = ({
   return (
     <section>
       <Helmet>
-        <title>{helmetTitle}</title>
+        <title>{title}</title>
       </Helmet>
 
-      <SearchForm sx={{ mb: '1rem' }} placeholder={t('tasks.search_placeholder')} />
+      <Typography variant="h2" sx={{ mb: '1rem' }}>
+        {title}
+      </Typography>
+
+      <SearchForm sx={{ mb: '1.8rem' }} placeholder={t('tasks.search_placeholder')} />
```

Comment:
noen grunn til at denne har blitt bumpet opp fra 1 til 1.8?
```suggestion
      <SearchForm sx={{ mb: '1rem' }} placeholder={t('tasks.search_placeholder')} />
```

Reply:
Det var pga tittelen, men den er nå flyttet.